src/compiler/passes/preprocess.js

```diff
@@ -0,0 +1,22 @@
+import util from '../utils/builder';
+
+export let createMethodHeaders = function(name, context) {
```

Comment:
I can sort of see why you put these in postprocess and preprocess, but they are really just utility functions that are used throughout.